### PR TITLE
Add --all support for view-last-applied 

### PR DESCRIPTION
**Pull request by shiywang**
Fixes https://github.com/kubernetes/kubernetes/issues/44195
sorry for my careless, and thanks for the review, @ymqytw ptal

### CODE REVIEW

**Comment by shiywang** [comment]
@k8s-ci-robot ok to test

**Comment by shiywang** [comment]
@k8s-bot non-cri node e2e test this

**Comment by shiywang** [comment]
@ymqytw ptal, thanks

**Comment by ymqytw** [comment]
Can you paste an example output of using --all to view multiple resources?

**Comment by shiywang** [comment]
@ymqytw sure, here is the output
```console
[root@shiywang template]# ./kubectl apply -f nginx.yaml 
deployment "nginx123" created
[root@shiywang template]# ./kubectl apply -f nginx-deployment.yaml 
deployment "nginx-deployment" created
[root@shiywang template]# ./kubectl apply view-last-applied deplopy --all
the server doesn't have a resource type "deplopy"
[root@shiywang template]# ./kubectl apply view-last-applied deploy --all
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  annotations: {}
  name: nginx-deployment
  namespace: default
spec:
  replicas: 3
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - image: nginx:1.7.9
        name: nginx
        ports:
        - containerPort: 80
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  annotations: {}
  name: nginx123
  namespace: default
spec:
  replicas: 4
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - image: gcr.io/google_containers/nginx-slim:0.8
        name: nginx
        ports:
        - containerPort: 80
[root@shiywang template]# 
```

**Comment by shiywang** [comment]
@adohe could you give me an approve ?

**Comment by shiywang** [comment]
@adohe kindly ping

**Comment by mengqiy** [comment]
@fabianofranz Need approval. Mind take a look. Thanks.

File: pkg/kubectl/cmd/apply_view_last_applied.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ type ViewLastAppliedOptions struct {
 	Selector                     string
 	LastAppliedConfigurationList []string
 	OutputFormat                 string
+	all                          bool
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/apply_view_last_applied.go]: Make it exported (e.g. `All`) to conform with the other attributes.

File: pkg/kubectl/cmd/apply_view_last_applied.go [hunk 2]
```diff
@@ -73,6 +74,7 @@ func NewCmdApplyViewLastApplied(f cmdutil.Factory, out, err io.Writer) *cobra.Co
 
 	cmd.Flags().StringP("output", "o", "", "Output format. Must be one of yaml|json")
 	cmd.Flags().StringVarP(&options.Selector, "selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.")
+	cmd.Flags().BoolVar(&options.all, "all", false, "[-all] to select all the specified resources.")
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/apply_view_last_applied.go]: Don't add "[-all]" to the flag description, make it just something like "select all resources of the specified resource types".

**Comment by shiywang** [comment]
@fabianofranz @mengqiy PTAL, thanks

File: pkg/kubectl/cmd/apply_view_last_applied.go (hunk 2 above)
Comment (mengqiy) [inline pkg/kubectl/cmd/apply_view_last_applied.go]: @shiywang Same usage is also in other commands' help text. Can you fix them?

**Comment by mengqiy** [comment]
@fabianofranz Any idea why `kubectl_stop.md` is not empty like other files in the same dir?
That file should be the same as others.

**Comment by shiywang** [comment]
@k8s-bot unit test this
@k8s-bot gci gce e2e test this

**Comment by shiywang** [comment]
@mengqiy https://github.com/kubernetes/kubernetes/issues/11384

**Comment by fabianofranz** [comment]
Thanks @shiywang!
/release-note-none
/approve
/lgtm

**Comment by fabianofranz** [comment]
@k8s-bot unit test this

**Pull request merged**